hack/00-osd-managed-cluster-config-integration.yaml.tmpl

```diff
@@ -1052,8 +1052,7 @@ objects:
         permissions:
         - allowFirst: true
           clusterRoleName: backplane-cssre-admins-project
-          namespacesAllowedRegex: .*
-          namespacesDeniedRegex: (^kube$|^kube-.*|^openshift$|^openshift-.*|^default$)
+          namespacesAllowedRegex: (^redhat-rhmi-.*|^redhat-rhoam-.*)
```

Comment:
this seems wrong

Reply:
Oh. This is from #545

Reply:
looks like this got fixed on master. rebasing and force-pushing a branch with just my changes (ie: excluding eb6a7c7)